internal/app/gateway/command/run.go

```diff
@@ -25,6 +27,7 @@ func init() {
 	gatewayCommand.PersistentFlags().BoolVarP(&dev, "dev", "d", false, "run mashling in dev mode")
 	gatewayCommand.PersistentFlags().StringVarP(&configCache, "config-cache", "C", ".cache", "location of the configuration artifacts cache")
 	gatewayCommand.PersistentFlags().BoolVarP(&configCacheEnabled, "config-cache-enabled", "E", true, "cache post-processed configuration artifacts locally")
+	gatewayCommand.PersistentFlags().StringVarP(&pingPort, "ping-port", "p", "9090", "configure mashling gateway ping service port")
```

Comment:
Adding the ping port here makes sense, but why add all the ping scaffolding in this command? Why not have the gateways themselves handle any services specific to it?